server/src/controllers/user.controller.js

```diff
@@ -28,14 +29,27 @@ const registerUser = async (fullname, username, password, email, status, lichess
     salt: salt,
     status: status
   });
+  const newAvatar = new Avatar({
```

Comment:
Please pass these values from the front-end only, and make the user-controller as loosely coupled as possible.